routers/rooms.py

```diff
@@ -81,14 +114,77 @@ async def delete_room(id: int):
 
 
 @router.get("/room", response_model=List[rooms_schema.RoomInfo])
-async def get_rooms(skip: int = 0, limit: int = 100):
+async def rooms_get(skip: int = 0, limit: int = 100):
     """
     Get list of rooms
-    
+
         Args:
                 skip (int): number for "offset" entries
                 limit (int): number for "limit" entries
         Returns:
                 List of dicts with rooms
     """
     return await rooms_utils.get_rooms(skip=skip, limit=limit)
+
+
+@router.get("/room/{room_number}/free")  # add response model
```

Comment:
```suggestion
@router.get("/rooms/{room_number}/status")  # add response model
```
You can return room status here. It could be `free`, `booked`, `needs_repairment` etc

Reply:
Done